roles/manage_cnp/tasks/add_function.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Validate cnpg_namespace
+  ansible.builtin.fail:
+    fail: msg="The variable 'cnpg_namespace' is not defined and empty"
```

Comment:
The `ansible.builtin.fail` module requires a `msg` attribute instead of the `fail` attribute. 
```
- name: Validate cnpg_namespace
  ansible.builtin.fail:
    msg: "The variable cnpg_namespace is empty"
  when: cnpg_namespace is defined and cnpg_namespace | length < 1
```
The fail message of the variable not being defined is also inconsistent with the conditionals defined.